docs/self-managed/identity/user-guide/configuration/connecting-to-entra-id.md

```diff
@@ -0,0 +1,45 @@
+---
+id: connecting-to-entra-id
+title: "Connecting to Entra ID (Azure AD)"
+description: "Learn how to connect your Camunda Self-Managed deployment to Entra ID (Azure AD)."
```

Comment:
I would expect to find this documentation in the "Installing on Microsoft AKS platform" section: https://docs.camunda.io/docs/next/self-managed/platform-deployment/helm-kubernetes/platforms/microsoft-aks/

Identity as a component is currently not involved with Azure at all, so we should also not hide this information there

Reply:
I have concerns about including this documentation in the helm section of our docs right now because these configuration variables are not natively supported in our helm charts currently. 

Although I agree regarding Identities hidden role here, if we look at the variable names "CAMUNDA_IDENTITY_ABC"  I think the Identity docs will be a natural place for readers to look. 

If we feel strongly about this though we could move it to the helm section but make a note that setting these variables requires the use of the direct env variables approach and not a specific param.

Reply:
The section is actually `Helm/Kubernetes`, so it is also about k8s installation and not just helm. I would not directly map `_IDENTITY_` to the C8 Identity as a product, it can also be part of the `Identity and Access Management` configuration on component level. I am open for other suggestions where this page could be located but I have a strong opinion that Identity user docs is definitely the wrong place, since it is not related to Identity as a product and it's users at all.

If I think about experience for anyone installing Camunda on an Azure cluster, then that person would most likely follow the installation described on the page I've linked.

Reply:
Ok, I've moved the page to this section now and also moved configuration to be more generalised here